calico/reference/public-cloud/gce.md

```diff
@@ -1,38 +1,27 @@
 ---
 title: Google Compute Engine
-description: Methods to ensure that traffic between containers on different hosts is not dropped by GCE fabric. 
+description: Methods to ensure that traffic between containers on different hosts is not dropped by GCE fabric.
 canonical_url: '/reference/public-cloud/gce'
 ---
 
-To deploy {{site.prodname}} in [Google Compute Engine (GCE)][GCE], you must ensure
-that traffic between containers on different hosts is not dropped by the GCE
-fabric. There are a few different options for doing this depending on your
-deployment.
+To deploy {{site.prodname}} in {% include open-new-window.html text='Google Compute Engine (GCE)' url='https://cloud.google.com/compute/' %}, you must ensure that traffic between containers on different hosts is not dropped by the GCE
+fabric.
+There are a few different options for doing this depending on your deployment.
 
 ## IP-in-IP encapsulation
 
-Container traffic routing can be enabled by setting [IP-in-IP encapsulation][IPIP]
-and NAT outgoing on the configured {{site.prodname}} IP pools.
+Container traffic routing can be enabled by setting [IP-in-IP encapsulation][{{ site.baseurl }}/networking/vxlan-ipip] and NAT outgoing on the configured {{site.prodname}} IP pools.
 
-See the [IP pool configuration reference][IPPool]
-for information on how to configure {{site.prodname}} IP pools.
+See the [IP pool configuration reference][{{ site.baseurl }}/reference/resources/ippool] for information on how to configure {{site.prodname}} IP pools.
```

Comment:
See the [IP pool configuration reference][{{ site.baseurl }}/reference/resources/ippool] to configure {{site.prodname}} IP pools.